### ISSUE TITLE
[FeatureFromSF] Automatic (scheduled) tour-import

### ISSUE THREAD

**Issue by wolfgang-ch** [mytourbook/mytourbook#773]
It would be a nice feature to import the same tour automaticly with a given schedule.

Motivation:
Imagine your doing the same tour ever workday, e.g. going to work by bike. With this feature you only have to log the tour once and it will be imported with a given schedule and options (tour-typ etc.)

Things to consider:
- there must be something like reference-tours wich will be imported
- one must be able to define one or more schedules e.g. like cron-job for the imports
- the timestamps of the imported positions must be corrected to the scheduled time
- Cause the programm is most likly not running at the scheduled time, the programm has to check for missed imports on startup or on user-request

Nice to have:
- before starting the import: show a list with the tours wich will be imported. Let the user select/deselect the tours which he want to import (e.g. maybe some days you haven't done the tour...)
- before starting the import: let the user change some properties of the imported tours (e.g. change the date-time the tour took place, change the time needed for the tour...20% faster...)

Thanks!

Reported by: mikerhosoft

Original Ticket: [mytourbook/feature-requests/121](https://sourceforge.net/p/mytourbook/feature-requests/121)
